src/controller/python/test/test_scripts/cluster_objects.py

```diff
@@ -210,6 +210,16 @@ async def TestReadAttributeRequests(cls, devCtrl):
         if res[1][Clusters.TestCluster][Clusters.TestCluster.Attributes.ListLongOctetString] != [b'0123456789abcdef' * 32] * 4:
             raise AssertionError("Unexpected read result")
 
+        logger.info("8: Read without fabric filter")
+        res = await devCtrl.ReadAttribute(nodeid=NODE_ID, attributes=[(1, Clusters.TestCluster.Attributes.ListFabricScoped)], fabricFiltered=False)
+        if len(res[1][Clusters.TestCluster][Clusters.TestCluster.Attributes.ListFabricScoped]) <= 1:
+            raise AssertionError("Expect more elements in the response")
+
+        logger.info("9: Read with fabric filter")
+        res = await devCtrl.ReadAttribute(nodeid=NODE_ID, attributes=[(1, Clusters.TestCluster.Attributes.ListFabricScoped)], fabricFiltered=True)
+        if len(res[1][Clusters.TestCluster][Clusters.TestCluster.Attributes.ListFabricScoped]) != 1:
+            raise AssertionError("Expect exact one element in the response")
```

Comment:
There should be a stronger check here - the fabricIndex of the element should match the accessing fabric index.